Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/PaddingTool.kt

```diff
@@ -0,0 +1,10 @@
+package de.rki.coronawarnapp.util
+
+object PaddingTool {
+    fun requestPadding(length: Int): String {
```

Comment:
did I miss the unit test?

Reply:
You did not, this needs one definitely one. Good point, will add :+1: